draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1343,47 +1349,54 @@ After the validation, the Verifier will have the following data for further proc
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4a - Verifiable Credentials with JSON payload based on the SD-JWT format
+## Example 4a - SD-JWT-based Verifiable Credentials (SD-JWT VC)
 
-In this example, the artifacts defined in this specification are used to secure
-Verifiable Credentials with JSON payload based on the SD-JWT format as defined in [@I-D.terbu-sd-jwt-vc]. Key Binding is applied
-using the Holder's public key passed in a `cnf` Claim in the SD-JWT.
+In this example, the artifacts defined in this specification are used to represent
+Verifiable Credentials with JSON payload as defined in [@I-D.terbu-sd-jwt-vc].
```

Comment:
maybe... ?

```suggestion
a verifiable credential with a JSON payload as defined in [@I-D.terbu-sd-jwt-vc].
```

Reply:
or was that supposed to be the title of terbu-sd-jwt-vc? It read kinda funny with plurals or lack thereof. Also the title has changed.  So maybe... ? 
```suggestion
SD-JWT-based Verifiable Credentials (SD-JWT VC) as defined in [@I-D.terbu-sd-jwt-vc].
```

Reply:
or title without the short part here: ... ?

 ```suggestion
SD-JWT-based Verifiable Credentials as defined in [@I-D.terbu-sd-jwt-vc].
```

Reply:
I like the middle option with the short part (yes, it was intended to be the doc title, thanks for catching!)